site/content/docs/5.0/components/navs.md

```diff
@@ -513,6 +513,18 @@ You can activate a tab or pill navigation without writing any JavaScript by simp
 </div>
 {{< /highlight >}}
 
+### Via jQuery
```

Comment:
Not sure if this should be here and not an `h4` after JavaScript, since jQuery belongs to JavaScript. /CC @mdo

Reply:
That makes a little more sense. I just thought that "Via JavaScript" implies pure JS.

Reply:
Yeah, let's move it below JavaScript. jQuery is JavaScript too.